src/main/resources/application.properties

```diff
@@ -38,6 +38,7 @@ waitlist.initialDelayResend: 2000
 waitlist.fixedRateResend: 15000
 database.name: eiffel_intelligence
 subscription.collection.name: subscription
+subscription.collection.repeatFlagHandlerName: subscription_repeat_handler
```

Comment:
what is this?

Reply:
What subscription(and which Requirement) has matched with which AggrObjId is stored in this new Collection in Db.